VMBackup/main/hostsnapshotter.py

```diff
@@ -180,7 +180,8 @@ def get_snapshot_info(self, responseBody):
                         except:
                             creationTimeObj = datetime.datetime.strptime(creationTimeString, "%Y-%m-%dT%H:%M:%SZ")
                         self.logger.log("Converting the creationTime string received in UTC format to UTC Ticks")
-                        timestamp = (creationTimeObj - epochTime).total_seconds()
+                        delta = creationTimeObj - epochTime
+                        timestamp = delta.days * 86400 + delta.seconds + delta.microseconds / 1e6
```

Comment:
can we create a helper method to getTotalSeconds and inside that method, add a check if total_seconds is unsupported then only use the backup method?

Reply:
also add a log statement for using the fallback logic